docs/developer/build_docker_image.md

```diff
@@ -0,0 +1,50 @@
+---
+sidebar_position: 1
+slug: /build_docker_image
+---
+
+# Build a RAGFlow Docker Image
+
+A guide explaining how to build a RAGFlow Docker image from its source code. By following this guide, you'll be equipped to create a local Docker image that can be used for development, debugging, or testing purposes.
+
+## Target Audience
+
+- Developers who have added new features or modified the existing code and require a Docker image to view and debug their changes.
+- Testers looking to explore the latest features of RAGFlow in a Docker image.
+
+## Prerequisites
+
+- CPU &ge; 4 cores
+- RAM &ge; 16 GB
+- Disk &ge; 50 GB
+- Docker &ge; 24.0.0 & Docker Compose &ge; v2.26.1
+
+:::tip NOTE
+If you have not installed Docker on your local machine (Windows, Mac, or Linux), see the [Install Docker Engine](https://docs.docker.com/engine/install/) guide.
+:::
+
+## Build a RAGFlow Docker Image
+
+To build a RAGFlow Docker image from source code:
+
+### 1. Git Clone the Repository
```

Comment:
Remove number?

Reply:
CONTRIBUTING.md will be moved to the root directory of the project.